# Msg that has a point cloud and a transformation 
std_msgs/Header header
uint32 id
sensor_msgs/PointCloud2 point_cloud
geometry_msgs/Transform T
bool hascloud
